maven-extension/src/main/java/io/opentelemetry/maven/handler/GoogleJibBuildHandler.java

```diff
@@ -29,6 +32,7 @@ public List<MavenGoal> getSupportedGoals() {
         MavenGoal.create("com.google.cloud.tools", "jib-maven-plugin", "build"));
   }
 
+  @SuppressWarnings("deprecation") // until old http semconv are dropped in 2.0
```

Comment:
@mateuszrzeszutek @jack-berg @laurit we haven't discussed bumping the contrib repo to 2.0 when we bump the instrumentation repo, but it could make sense to keep the -instrumentation and -contrib repos version aligned

@srprash @wangzlei would you want to follow this approach instead of the clean break that was merged in #1050? (we have time before the next release to revert that change if you want)